rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -8,30 +8,29 @@
 
```

Comment:
Can you avoid committing to gameplay_node in this PR? You can reset this file to what's in ros2 with https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch